code/Makefile

```diff
@@ -371,6 +379,16 @@ $(filter %$(STABILIZE_E_SUFFIX), $(STABILIZE_WEBSITE)): %$(STABILIZE_E_SUFFIX):
 $(filter %$(TEX_E_SUFFIX), $(TEX_EXAMPLES)): %$(TEX_E_SUFFIX): %$(TRACE_GRAPH_SUFFIX)
 	EDIR="$(EDIR)" BUILD_FOLDER="$(BUILD_FOLDER)" SUMMARIZE_TEX=$(SUMMARIZE_TEX) MAKE="$(MAKE)" TERM="$(BATCHTERM)" "$(SHELL)" "$(SCRIPT_FOLDER)"tex_build.sh
 
+# Build individual example mdBook SRSs.
+# Targets will have the form exampleName_mdBook_build.
+$(filter %$(MDBOOK_BUILD_E_SUFFIX), $(MDBOOK_BUILD_EXAMPLES)): %$(MDBOOK_BUILD_E_SUFFIX): $(CLEAN_GF_PREFIX)$(LOG_FOLDER_NAME) %$(GEN_E_SUFFIX)
+	- mdbook build "$(BUILD_FOLDER)$(EDIR)/SRS/mdBook"
+
+# Launch local server for individual example mdBook SRSs.
+# Targets will have the form exampleName_mdBook_server.
+$(filter %$(MDBOOK_SERVER_E_SUFFIX), $(MDBOOK_SERVER_EXAMPLES)): %$(MDBOOK_SERVER_E_SUFFIX): $(CLEAN_GF_PREFIX)$(LOG_FOLDER_NAME) %$(GEN_E_SUFFIX)
+	- mdbook serve --open "$(BUILD_FOLDER)$(EDIR)/SRS/mdBook"
```

Comment:
The `-` in front suppresses error codes. I think we want those to be returned (I previously added too many of the `-` in the Makefile, I need to fix those too). So, we should remove the prepended `-`s.

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3823/commits/6eaf3dccd8fd7da4b86aa5df17a5bfd496897c68.